content/articles/terraform-provider.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: Terraform Provider
+excerpt: How to use DNSimple's Terraform provider to manage your DNS
+categories:
+- Services
+- Terraform and DNSimple
+---
+
+# DNSimple Terraform Provider
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+[Terraform](https://terraform.io) is an Infrastructure as Code (IoC) tool that allows you to build, change, and version
+infrastructure safely and efficiently. With [DNSimple's Terraform Provider](https://registry.terraform.io/providers/dnsimple/dnsimple/latest),
+you can easily manage your DNS infrastructure and set up the required DNS records.
+
+## Configuring the DNSimple provider
+
+```
```

Comment:
I wish we had syntax highlighting!

Reply:
We do have the gem for it: `kramdown-parser-gfm` but it's true that it's not working at the moment. I'll open an issue to track this down.

Reply:
Nevermind, it was nanoc not refreshing the page correctly, but I got it to work:
<img width="289" alt="Screen Shot 2021-11-30 at 10 02 56 AM" src="https://user-images.githubusercontent.com/80610/144017183-517bd13d-e42c-49dc-a3e1-f259bd95c368.png">

Here's how I did that

<img width="358" alt="Screen Shot 2021-11-30 at 10 03 50 AM" src="https://user-images.githubusercontent.com/80610/144017218-4b843932-7bd2-4041-8b9f-e82ca084059d.png">

However, it's true that the washed out red background is annoying. I'll fix that later.